extensions/sendbird/v1/actions/createTicket/config/fields.ts

```diff
@@ -23,6 +24,14 @@ export const fields = {
     description:
       'Specifies the title of a ticket, which will be the group channel name in Sendbird Chat platform as well. Maximum length is 100 characters.',
   },
+  channelUrls: {
```

Comment:
@michal-grzelak Let's call this `relatedChannelUrls` so it matches 1-to-1 with Sendbird. Also change the label to `Related channel URLs`.